addons/popochiu/engine/objects/gui/components/dialog_text/dialog_text.gd

```diff
@@ -73,14 +73,18 @@ func play_text(props: Dictionary) -> void:
 	
 	if PopochiuConfig.should_talk_gibberish():
 		msg = D.create_gibberish(msg)
-	
+
 	# Call the virtual method that modifies the size of the RichTextLabel in case the dialog style
 	# requires it.
 	await _modify_size(msg, props.position)
 	
+	rich_text_label.push_color(props.color)
```

Comment:
We removed this in #357 because it was causing issues when centering text. Please remove it again since each **DialogText** component should assign the color itself using BBCode (`[color][/color]`).